dynamic.js

```diff
@@ -27,7 +27,8 @@ module.exports = function (fastify, opts, next) {
   const security = opts.swagger.security || null
 
   if (opts.exposeRoute === true) {
-    fastify.register(require('./routes'))
+    const prefix = opts.routePrefix || '/api-docs'
+    fastify.register(require('./routes'), { prefix })
```

Comment:
@mcollina 
I did not have time today to dive in why this one is not working. but for some reason when passing `{ prefix: '/api-docs' }` tests are failing and I had to manually use the `opts.prefix`

Reply:
I was expecting that the `prefix` attribute will automatically add a prefix and I wouldn't need to add `opts.prefix` to the routes.

Reply:
I think I got it, I was expecting that `prefix` will be used inside of a plugin (same as a route).